manifest.webapp

```diff
@@ -38,7 +38,7 @@
     "settings": {
       "description": "Required by the cozy-bar to display Claudy and know which applications are coming soon",
       "type": "io.cozy.settings",
-      "verbs": ["GET"]
+      "verbs": ["GET", "PUT"]
```

Comment:
FYI, when you edit the permissions of an app, you should handle the fact that the user should update the app by itself. Cozy-Stack will block the automatic upgrade process if the permissions has changed, because the user has accepted the previous permissions and not the new ones. EXECEPT for what we call `platformApps`. Those platformApps can edit their permissions and be upgraded automacily without any user consent. 

Here is the list: https://github.com/cozy/cozy-stack/blob/7e0e0c796a7f8c57ca6a128258be1af77421da27/pkg/consts/consts.go#L100

Pass is already on it 👍

Reply:
`you should handle the fact that the user should update the app by itself `
How would it be handled (if the app was not in the list) ? Is there a specific API to call?
Why isn't this handled by the stack? (ex: automatique `upgrade with new permissions` popup displayed to the user when he log-in)

Reply:
My bad you're right, the stack handles that case and display a page with a message saying that you HAVE TO upgrade to get the app running with a CTA to the store. You can't use the app anymore without upgrading. 

This should be different for an OAuth Client linked to a webapp (https://docs.cozy.io/en/cozy-stack/auth/#linked-applications). The client will not have the new permission if he doesn't install the new version on the cozy, so the app will have to handle that case (not sure I'm clear enough). 

It's also different for a konnector. If a konnector has new permissions, it can run on the previous version until the user accepts the new one. 

Thank you for forcing me to dig into this point again. This is not a common use case for now but we should keep that in mind

Reply:
> Thank you for forcing me to dig into this point again

*cough*
hmm yeah, this was totally intended... you're welcome 😅